### PR TITLE
fix links from screenshots to web console documentation

### PR DESCRIPTION
**Pull request by winne42**
hope this works :-)

### CODE REVIEW

File: docs/develop/connect.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ this will be `http://localhost:9000/`.
 
 import Screenshot from "@theme/Screenshot"
 
-<a href="web-console">
+<a href="/docs/develop/web-console">
```
Comment (amyshwang) [inline docs/develop/connect.md:33]:
```suggestion
<a href="/docs/develop/web-console/">
```

File: docs/develop/query-data.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ QuestDB ships with an embedded Web Console running by default on port `9000`.
 
 import Screenshot from "@theme/Screenshot"
 
-<a href="web-console">
+<a href="/docs/develop/web-console">
```
Comment (amyshwang) [inline docs/develop/query-data.md:33]:
```suggestion
<a href="/docs/develop/web-console/">
```

**Review by amyshwang — CHANGES_REQUESTED** [review]
Thank you for your contribution @winne42! Very minor suggestions added as we're trying to be consistent with using trailing slash for hyperlinks.

**Comment by winne42** [comment]
> Thank you for your contribution @winne42! Very minor suggestions added as we're trying to be consistent with using trailing slash for hyperlinks.

Allright, thanks!

**Review by amyshwang — APPROVED** [review]
Thank you!

**Pull request merged**